src/policy/immix/block.rs

```diff
@@ -254,10 +260,42 @@ impl Block {
                 mark_histogram[holes] += marked_lines;
                 // Record number of holes in block side metadata.
                 self.set_holes(holes);
+
+                #[cfg(feature = "vo_bit")]
+                vo_bit::helper::on_region_swept::<VM, _>(self, true);
+
                 false
             }
         }
     }
+
+    /// Clear VO bits metadata for regions that contain young objects.
+    #[cfg(feature = "vo_bit")]
+    pub fn clear_vo_bits_for_young_regions(&self, line_mark_state: Option<u8>) {
```

Comment:
You can just rename this as `clear_vo_bits_for_unmarked_region`. There is no need to mention young/old.

Reply:
Done.